keras_cv/utils/preprocessing.py

```diff
@@ -16,6 +16,38 @@
 
 from keras_cv import core
 
+ResizeMethod = tf.image.ResizeMethod
```

Comment:
do we really need this since tf.image.ResizeMethod is already string based?
https://github.com/tensorflow/tensorflow/blob/v2.10.0/tensorflow/python/ops/image_ops_impl.py#L1419-L1429
Having less indirection would be probably more preferrable.

Reply:
+1 -- if possible to remove this I think that would be easier to follow

Reply:
Done